src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -57,23 +63,53 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         this.props.onVersionChange();
     }
 
-    fetchFile = () => {
-        const { api, fileId } = this.props;
+    handleActionDelete = (versionId: string): void => {
+        this.setState({ isLoading: true }, () => {
+            this.deleteVersion(versionId)
+                .then(this.fetchData)
+                .then(this.handleDeleteSuccess)
```

Comment:
When this is called after `fetchData`, isn't the argument `[BoxItem, FileVersions]`? What does this first invocation of `handleDeleteSuccess` do?

Reply:
Looks like another artifact of the bad rebase. I'll go through the patchset again carefully to make sure there's no other issues.